pom.xml

```diff
@@ -29,7 +29,7 @@
 		<!-- cryptomator dependencies -->
 		<cryptomator.cryptolib.version>2.1.0-rc1</cryptomator.cryptolib.version>
 		<cryptomator.cryptofs.version>2.4.4</cryptomator.cryptofs.version>
-		<cryptomator.integrations.version>1.1.0</cryptomator.integrations.version>
+		<cryptomator.integrations.version>1.2.0</cryptomator.integrations.version>
```

Comment:
Use 1.2.0-beta1 for now, so we can already merge this even before 1.2.0 final is ready.

Reply:
updated